python/pyngraph/function.cpp

```diff
@@ -49,4 +51,24 @@ void regclass_pyngraph_Function(py::module m)
             py::cast(self.get_output_shape(0)).attr("__str__")().cast<std::string>();
         return "<" + class_name + ": '" + self.get_friendly_name() + "' (" + shape + ")>";
     });
+    function.def_static("from_capsule", [](py::object* capsule) {
+        auto* pybind_capsule_ptr = capsule->ptr();
+        auto* capsule_ptr = PyCapsule_GetPointer(pybind_capsule_ptr, CAPSULE_NAME);
```

Comment:
Please add a one-line comment to explain what `PyCapsule_GetPointer` does and where it comes from.
Shouldn't we include `Python.h` to use this?

Reply:
I does come from `Python.h` but this header is already included by `pybind11.h`